File: eval/fixtures/legacy-directory-backed-aspnet-commerce/src/Legacy/Customers/CustomerProfileRepository.cs
namespace Acme.Legacy.Customers;

public class CustomerProfileRepository
{
  public CustomerRiskResult LoadProfileRiskSnapshot(string customerId)
  {
    var sql = "SELECT * FROM customers.aspnet_customer_profile_snapshots WHERE customer_id = @customerId";
    return new CustomerRiskResult(sql);
  }
}

File: eval/fixtures/legacy-directory-backed-struts-commerce/src/main/java/com/acme/legacy/customers/CustomerProfileRepository.java
package com.acme.legacy.customers;

public class CustomerProfileRepository {
  public CustomerRiskResult loadProfileRiskSnapshot(String customerId) {
    String sql = "SELECT * FROM customers.struts_customer_profile_snapshots WHERE customer_id = ?";
    return new CustomerRiskResult(sql);
  }
}
